### PR TITLE
Add GitHub workflow to pack BI connectors

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
### Description
Renamed Power BI connector as "OpenSearch Project"
Added a new GitHub workflow to build (pack) BI connectors, steps are:
- pack Tableau JDBC connector
- ~~pack Tableau ODBC connector~~
- pack Power BI connector for 'OpenSearch Project'
- apply some changes to Power BI connector to rename it
- pack the Power BI connector for 'Amazon OpenSearch Service'
- publish artifacts

https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/1593959072
 
### Issues Resolved
AOS-253
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by guiangumpac — APPROVED** [review]

File: bi-connectors/PowerBIConnector/SqlOdbcPBIConnector.pq [hunk 1]
```diff
@@ -19,7 +19,7 @@ SqlOdbcPBIConnectorType = type function (
         ])
     )
     as table meta [
-        Documentation.Name = "OpenSearch"
+        Documentation.Name = "OpenSearch Project"
```
Comment (kylepbit) [inline bi-connectors/PowerBIConnector/SqlOdbcPBIConnector.pq:22]: Where are these changes here, they seem unrelated to the GitHub flows?

File: .github/workflows/bi-connectors.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      - 'bi-connectors/PowerBIConnector/**'
+      - 'bi-connectors/TableauConnector/**'
+      - '.github/workflows/bi-connectors.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Pack Tableau ODBC connector
```
Comment (kylepbit) [inline .github/workflows/bi-connectors.yml:15]: Why do we pack both ODBC and JDBC? I thought we were only going to use JDBC going forward?

File: .github/workflows/bi-connectors.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    - name: Prepare Power BI ODBC connector
+      run: |
+        cp SqlOdbcPBIConnector.pq SqlOdbcPBIConnector.m
+      working-directory: bi-connectors/PowerBIConnector
+    - name: Pack Power BI ODBC connector
+      id: pack-powerbi-odbc-os-proj
+      run: |
+        zip OpenSearchProject.mez *.png *.m *.resx *.pqm
+      working-directory: bi-connectors/PowerBIConnector
+    - name: Prepare the second Power BI ODBC connector
```
Comment (kylepbit) [inline .github/workflows/bi-connectors.yml:34]: What is the second one...?

Comment (Yury-Fridlyand) [reply .github/workflows/bi-connectors.yml:34]: There are 2 connector requested by Amazon: one for `OpenSearch Project` which is being build from the repo as is; and the second one - for `Amazon OpenSearch Service`. To build the second one we need to modify few strings in the connector code.

File: .github/workflows/bi-connectors.yml (hunk 2 above)
Comment (Yury-Fridlyand) [reply .github/workflows/bi-connectors.yml:15]: This script build both for future reference. At the moment, repo already contains a taco for ODBC, so nothing changed for ODBC actually.

File: bi-connectors/PowerBIConnector/SqlOdbcPBIConnector.pq (hunk 1 above)
Comment (Yury-Fridlyand) [reply bi-connectors/PowerBIConnector/SqlOdbcPBIConnector.pq:22]: According to Amazon's request we should build 2 connectors: one for `OpenSearch Project` and the second one - for `Amazon OpenSearch Service`. `OpenSearch` is not a valid name.

File: .github/workflows/bi-connectors.yml (hunk 3 above)
Comment (kylepbit) [reply .github/workflows/bi-connectors.yml:34]: Can you then name them accordingly instead of "the second one"?

File: .github/workflows/bi-connectors.yml (hunk 2 above)
Comment (kylepbit) [reply .github/workflows/bi-connectors.yml:15]: The ODBC one is now deprecated for the JDBC one, we should stop building the ODBC one.

File: .github/workflows/bi-connectors.yml (hunk 3 above)
Comment (Yury-Fridlyand) [reply .github/workflows/bi-connectors.yml]: Changed in 6e378705.

File: .github/workflows/bi-connectors.yml (hunk 2 above)
Comment (Yury-Fridlyand) [reply .github/workflows/bi-connectors.yml]: Disabled in 4cadfd18.

File: .github/workflows/bi-connectors.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    steps:
+    - uses: actions/checkout@v1
+    - name: Pack Tableau JDBC connector
+      id: pack-tableau-jdbc
+      run: |
+        zip -r opensearch_sql_jdbc.taco . -x *.taco
+      working-directory: bi-connectors/TableauConnector/opensearch_sql_jdbc
+    - name: Prepare Power BI ODBC connector
+      run: |
+        cp SqlOdbcPBIConnector.pq SqlOdbcPBIConnector.m
```
Comment (kylepbit) [inline .github/workflows/bi-connectors.yml:22]: Are these names right after the rebranding that occurred?

**Review by kylepbit — APPROVED** [review]
